openedx/stanford/cms/djangoapps/contentstore/tests/test_bulkupdate.py

```diff
@@ -0,0 +1,94 @@
+"""
+Unit tests for bulk update problem settings utility
+
+TODO Write unit tests for bulk_update_problem_settings from utilities.tasks
```

Comment:
Did you manage to get the tests cleaned up?

Reply:
Resolved the issue from our convo f2f (used `@mock.patch` as a workaround) --- I've written tests for checking advanced settings and problem settings are modified, but not edge cases, exceptions, etc., yet